.github/workflows/orca.yml

```diff
@@ -68,13 +68,13 @@ jobs:
       - name: Install
         run: |
           ../orca/bin/ci/install.sh
-          ./bin/travis/install.sh
+          ./.github/install.sh
```

Comment:
The scripts in `./bin/travis` contain relative paths that need to be changed if we're going to change their directory depth. If you look at the static analysis job log, for example, you'll see this:

```
# Before script
./.github/install.sh: line 15: ../../../orca/bin/travis/_includes.sh: No such file or directory

# Script
./.github/script.sh: line 15: ../../../orca/bin/travis/_includes.sh: No such file or directory
./.github/script.sh: line 21: ../../../orca/bin/travis/script.sh: No such file or directory
```

Reply:
Well that's not good that tests didn't fail on that error. I fixed it in the next commit and opened an upstream PR: https://github.com/acquia/orca/pull/182